client/configuration/config.js

```diff
@@ -1,5 +1,5 @@
 angular.module('config', [])
 
-.constant('ENV', {RequiresAuth:false,etcdHost:'127.0.0.1',etcdPort:'4001',hobknobHost:'localhost',hobknobPort:'3006',categories:[{id:0,name:'Simple Features',description:'Use when you want your feature to be either on or off'},{id:1,name:'Domain Features',description:'Use when you want your features to be toggled separately for different domains (e.g. com, couk, fr, ...)',values:['com','couk','de','fr']},{id:2,name:'Locale Features',description:'Use when you want your features to be toggled separately for different locales (e.g. en-GB, en-US, fr-FR, ...)',values:['en-GB','en-US','fr-FR','de-DE']}],plugin:{path:'../server/examplePlugin.js'}})
```

Comment:
thanks for pointing this out, i've missed the grunt tasks there. I've updated the code to make use of this file. 
One thing that will happen then, is that on client side we will have the ClientSecret key, maybe we should have a task to remove the not need keys from client/config.js